Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:16.04
+
+ADD . /app
+
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+RUN apt-get update
+RUN apt-get build-dep -y python-imaging
+RUN apt-get install -y python3-pip \
+                       libjpeg8 \
+                       libjpeg62-dev \
+                       libfreetype6 \
+                       libfreetype6-dev \
+                       wget
+
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 4.2.0
+
+RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.4/install.sh | bash && \
+    source $NVM_DIR/nvm.sh && \
+    nvm install $NODE_VERSION && \
+    nvm alias default $NODE_VERSION && \
+    nvm use default
+
+ENV NODE_PATH $NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
+WORKDIR /app
+
+RUN pip3 install --upgrade pip
+RUN pip3 install -r requirements/dev.txt
```

Comment:
The reason to add this here is to make creating dev environment in the fastest way we can. So I prefer to leave dependecies insallation here (so it will not be chekced every time user run container).

When somebody will stay in project for longer period of time, he/she will able to do it manually.